scripts/setup/requirements.silabs_docker.txt

```diff
@@ -0,0 +1,10 @@
+# The below dependencies are being installed by apt inside docker
```

Comment:
why in scripts/setup instead inside of `chip-build-efr32` ?

Maybe consider having this file and then copy inside docker and `pip install -r` on it, so it is always up to date.

Reply:
I see silabs_docker is indeed a platform in .gitmodules (am surprised since we already have efr32) ... do you happen to know the history behind that?

Reply:
Not that I know completely but the use of it seems to indicate the repos we want to clone inside docker, while the rest can be used for a "persistent" build environment.

Reply:
@andy31415 We added `silabs_docker` to avoid cloning the `gecko_sdk` everytime the ci sets up an environnement.
`efr32` clones the `gecko_sdk` submodule while `silabs_docker` does not and uses the pre-installed one.

Reply:
I think checking in a file with all comments does not look as useful.

It is likely easy to get out of sync with the dockerfile anyway. I would prefer to completely remove this file.